src/wrapper/commons.jl

```diff
@@ -700,7 +700,7 @@ const SCIP_PRIMAL = SCIP_Primal
 # Skipping MacroDefinition: SCIP_DECL_PROBEXITSOL ( x ) SCIP_RETCODE x ( SCIP * scip , SCIP_PROBDATA * probdata , SCIP_Bool restart )
 # Skipping MacroDefinition: SCIP_DECL_PROBCOPY ( x ) SCIP_RETCODE x ( SCIP * scip , SCIP * sourcescip , SCIP_PROBDATA * sourcedata , SCIP_HASHMAP * varmap , SCIP_HASHMAP * consmap , SCIP_PROBDATA * * targetdata , SCIP_Bool global , SCIP_RESULT * result )
 
-@cenum(SCIP_Objsense{Int32},
+@cenum(SCIP_Objsense,
```

Comment:
All of this code is auto-generated and should not be edited.

Reply:
SCIP_Objsense{Int32} type is "Expr" 

CEnum.jl
 77     elseif !isa(T, Symbol)  ==> ERROR
 78         throw(ArgumentError("invalid type expression for Cenum $T"))
 79     end